src/lib.rs

```diff
@@ -25,8 +25,10 @@ pub struct Builder {
     includes: Vec<String>,
     black_list: Vec<String>,
     out_dir: String,
+    #[allow(dead_code)]
```

Comment:
I'm not sure what these dead fields are for. Maybe a todo, I think. So I just add an allow attribute

Reply:
I think they are used, they can be set via public API and are passed on, at least when using Prost. Maybe they can be cfg'ed rather than allowing dead code?